lib/mumukit/platform/organization/helpers.rb

```diff
@@ -0,0 +1,73 @@
+
+module Mumukit::Platform::Organization::Helpers
+  extend ActiveSupport::Concern
+
+  included do
+    delegate :logo_url,
+             :theme_stylesheet_url,
+             :extension_javascript_url, to: :theme
+
+    delegate :login_methods,
+             :login_methods=,
+             :login_settings,
+             :raise_hand_enabled?,
+             :raise_hand_enabled=,
+             :public?,
+             :public=,
+             :private?, to: :settings
+
+    delegate :locale,
```

Comment:
Please move locale to settings, logo_url to community, and rename community to profle.